### PR TITLE
Add Storybook with Visual testing

### PR DESCRIPTION
**Pull request by aspirisen**

### CODE REVIEW

**Review by oleg-semyonov — COMMENTED** [review]
I'm a bit confused on how this should be used in CI/QA process.
I see that the `storybook` action creates static files generated by loki, but how would those be checked in pull request?

**Comment by aspirisen** [comment]
@oleg-semyonov loki runs headless chrome for taking screenshots
> storybook action creates static files generated by loki

loki doesn't generate static files, they are generated by storybook, then loki uses this static files in the browser

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,7 @@ node_modules/
 tsconfig.tsbuildinfo
 package-lock.json
 storybook-static
+.loki/current
+.loki/difference
+.loki/.gitignore
```
Comment (oleg-semyonov) [inline .gitignore:11]: This is weird, why would you want to ignore a `.gitignore` file? 🤔

File: README.md [hunk 2]
```diff
@@ -86,10 +85,20 @@ npm install
...
 ```
 
-----
+## Loki
 
-Brought to you by [Prezly][prezly].
+We use [Loki](https://loki.js.org/) visual testing library for storybook.
+There are several commands that you can use:
+
+1. `loki` - checks that your current stories are matched what you already have (Storybook server should run)
```
Comment (oleg-semyonov) [inline README.md:93]:
```suggestion
1. `loki` - checks that your current stories are matched with what you already have (Storybook server should run)
```

File: README.md [hunk 3]
```diff
@@ -86,10 +85,20 @@ npm install
...
+1. `loki:update` - Regenerate all snapshots despite what you already have (Storybook server should run)
+1. `loki:static-build` - Performs checks like `loki` does, but over builded sources from storybook, no need to run Storybook server
 
+If Loki fails on CI you can download generated images like [here](https://github.com/actions/upload-artifact#where-does-the-upload-go)
```
Comment (oleg-semyonov) [inline README.md:97]: Indeed, where does it go? 😄

File: src/@types/lokie.d.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+declare module '@loki/create-async-callback' {
```
Comment (oleg-semyonov) [inline src/@types/lokie.d.ts:1]: There is a typo in this file's name ;)
I guess it should be `loki.d.ts`

File: src/elements/Bookmark/Bookmark.slate.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+    title: 'Elements/Bookmark',
+    decorators: [
+        (Story) => {
+            const cb = createAsyncCallback();
+
+            setTimeout(() => {
+                cb();
+            }, 1000);
+
+            return <Story />;
```
Comment (oleg-semyonov) [inline src/elements/Bookmark/Bookmark.slate.stories.tsx:16]: Looks weird. What is it supposed to do exactly? 🤔 
Does it simulate a 1 second delay before Story is returned?

File: .gitignore (hunk 1 above)
Comment (aspirisen) [reply .gitignore:11]: When you run loki it automatically creates gitignore file, what is not necessary

File: README.md (hunk 3 above)
Comment (aspirisen) [reply README.md:97]: It creates zip file which you can download on actions page like in the example here https://github.com/actions/upload-artifact#where-does-the-upload-go

File: src/elements/Bookmark/Bookmark.slate.stories.tsx (hunk 5 above)
Comment (aspirisen) [reply src/elements/Bookmark/Bookmark.slate.stories.tsx:16]: It delays 1s before screenshot capture, not the story render

**Review by e1himself — APPROVED** [review]

**Pull request merged**